<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>propHistogram</class>
 <widget class="QWidget" name="propHistogram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>332</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>332</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>332</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="histogram">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>0</y>
     <width>194</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Histogram</string>
   </property>
   <widget class="QDoubleSpinBox" name="upperLimit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>34</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>-999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="lowerLimit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>14</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 127, 0);</string>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>-999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="levels">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>64</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1024</number>
    </property>
    <property name="value">
     <number>512</number>
    </property>
   </widget>
   <widget class="QLabel" name="sizeLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>66</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Levels: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>18</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower Limit:</string>
    </property>
   </widget>
   <widget class="QLabel" name="nLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>37</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Upper Limit: </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="colorSchemeCombo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>89</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Color scheme: </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="absCheckBox">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>65</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>abs</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lowerLimit</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propHistogram</receiver>
   <slot>changedLowerLimit(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upperLimit</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propHistogram</receiver>
   <slot>changedUpperLimit(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>levels</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propHistogram</receiver>
   <slot>changedLevels(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorSchemeCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>propHistogram</receiver>
   <slot>colorSchemeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>absCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>propHistogram</receiver>
   <slot>changedABS(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changedLevels(int)</slot>
  <slot>changedLowerLimit(double)</slot>
  <slot>changedUpperLimit(double)</slot>
  <slot>colorSchemeChanged(int)</slot>
  <slot>changedABS(int)</slot>
 </slots>
</ui>
